src/Pos/Communication/Limits.hs

```diff
@@ -105,9 +111,9 @@ instance MessageLimited SecretSharingExtra where
         return $ SecretSharingExtra <$> msgLenLimit <+> vector numLimit
 
 instance MessageLimited (AsBinary SecretSharingExtra) where
-    -- 20 is a magic constant copy-pasted from 'MessageLimitedPure' instance
     getMsgLenLimit _ =
-        coerce . (20 +) <$> getMsgLenLimit (Proxy @SecretSharingExtra)
+        coerce . (maxAsBinaryOverhead +) <$>
```

Comment:
Wow, now I realize what this example means in `Data.Coerce` module
`instance Coercible b b' => Coercible (D a b c) (D a b' c')`